ext/vulnsrc/manual/manual.go

```diff
@@ -1514,6 +1514,388 @@ var Vulnerabilities = []database.Vulnerability{
+		Description: "CUPS cups-browsed before 2.5b1 will send an HTTP POST request to an arbitrary destination and port in response to a single IPP UDP packet requesting a printer to be added, a different vulnerability than CVE-2024-47176. (The request is meant to probe the new printer but can be used to create DDoS amplification attacks.)",
+		Link:        "https://security-tracker.debian.org/tracker/CVE-2024-47850",
+		// Debian did not assign a severity, so basing this on the score.
+		Severity: database.HighSeverity,
+		FixedIn: []database.FeatureVersion{
+			{
+				Feature: database.Feature{
+					Name: "cups-filters",
+					Namespace: database.Namespace{
+						Name:          "debian:12",
+						VersionFormat: dpkg.ParserName,
+					},
+				},
+				Version: versionfmt.MaxVersion,
+			},
+		},
+		Metadata: map[string]interface{}{
+			"NVD": map[string]interface{}{
+				"PublishedDateTime":    "2024-10-04T16:00Z",
+				"LastModifiedDateTime": "2024-10-07T16:00Z",
+				"CVSSv2": map[string]interface{}{
+					"ExploitabilityScore": 0.0,
+					"ImpactScore":         0.0,
+					"Score":               0.0,
+					"Vectors":             "",
+				},
+				"CVSSv3": map[string]interface{}{
+					"ExploitabilityScore": 3.9,
+					"ImpactScore":         3.6,
+					"Score":               7.5,
+					"Vectors":             "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
+				},
+			},
+		},
+	},
+	// Debian unstable
+	{
+		Name: "CVE-2024-47850",
+		Namespace: database.Namespace{
+			Name:          "debian:unstable",
+			VersionFormat: dpkg.ParserName,
+		},
+		Description: "CUPS cups-browsed before 2.5b1 will send an HTTP POST request to an arbitrary destination and port in response to a single IPP UDP packet requesting a printer to be added, a different vulnerability than CVE-2024-47176. (The request is meant to probe the new printer but can be used to create DDoS amplification attacks.)",
+		Link:        "https://security-tracker.debian.org/tracker/CVE-2024-47850",
+		// Debian did not assign a severity, so basing this on the score.
+		Severity: database.HighSeverity,
+		FixedIn: []database.FeatureVersion{
+			{
+				Feature: database.Feature{
+					Name: "cups-filters",
+					Namespace: database.Namespace{
+						Name:          "debian:unstable",
+						VersionFormat: dpkg.ParserName,
+					},
+				},
+				Version: versionfmt.MaxVersion,
+			},
+		},
+		Metadata: map[string]interface{}{
+			"NVD": map[string]interface{}{
+				"PublishedDateTime":    "2024-10-04T16:00Z",
+				"LastModifiedDateTime": "2024-10-07T16:00Z",
+				"CVSSv2": map[string]interface{}{
+					"ExploitabilityScore": 0.0,
+					"ImpactScore":         0.0,
+					"Score":               0.0,
+					"Vectors":             "",
+				},
+				"CVSSv3": map[string]interface{}{
+					"ExploitabilityScore": 3.9,
+					"ImpactScore":         3.6,
+					"Score":               7.5,
+					"Vectors":             "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
+				},
+			},
+		},
+	},
+	// Ubuntu 16.04
```

Comment:
Do you mind also updating the fixed in version for https://ubuntu.com/security/CVE-2024-47175 and https://ubuntu.com/security/CVE-2024-47176 ? sorry 😬

Reply:
Also updated the scores for to match Ubuntus: https://ubuntu.com/security/CVE-2024-47176#impact-score

- Previous test had base score of 8.3
- Ubuntu page has base score of 7.5
- NVD has base score of 5.3